homework-1/src/repository/Calculator.java

```diff
@@ -0,0 +1,50 @@
+package repository;
+
+import java.util.Scanner;
+
+public class Calculator {
+	/*
+	 * Functia rezolvare primeste doua numere si o operatie si in functie de
+	 * operatie, alege ce face cu acele doua numere
+	 */
+	public double rezolvare(double numar, char operatie, double numar2) {

```

Comment:
naming not ok "rezolvare"
the function does too may things, you should have separate method for all operations